src/redux/reducers/rootReducer.ts

```diff
@@ -0,0 +1,12 @@
+import { combineReducers } from '@reduxjs/toolkit';
+import api from './api';
+import mainReducer from './mainReducer';
+import formReducer from './formReducer';
+
+const rootReducer = combineReducers({
+  main: mainReducer.reducer,
+  api: api.reducer,
```

Comment:
For redux toolkit query you can use in such way: [api.reducerPath]:api.reducer. And we can use createStore and configureStore instead of combineReducers. You can read about this in docs.